### PR TITLE
Kubernetes doc updates

### PR DESCRIPTION
**Pull request by mythi**

### CODE REVIEW

File: source/clear-linux/tutorials/kubernetes/kubernetes.rst [hunk 1]
```diff
@@ -150,45 +150,37 @@ Configure and run CRI-O + kata-runtime
        sudo systemctl daemon-reload
        sudo systemctl restart crio
 
-#.  Initialize the master control plane with the command:
+#.  Initialize the master control plane with the command below and follow the printed instructions to set up `kubectl`:
```
Comment (MCamp859) [inline source/clear-linux/tutorials/kubernetes/kubernetes.rst]: Suggest changing from "follow the printed" to "follow the displayed".

**Comment by mvincerx** [comment]
I'll merge this now. @MCamp859 you can follow up on a TCS Editorial review PR for more edits.

File: source/clear-linux/tutorials/kubernetes/kubernetes.rst (hunk 1 above)
Comment (mythi) [reply source/clear-linux/tutorials/kubernetes/kubernetes.rst]: done.

**Comment by mythi** [comment]
@mvincerx updated

**Comment by mvincerx** [comment]
Thanks, all. LGTM.

**Pull request merged**